### ISSUE TITLE
Differences in Omega_n_rel between CCL and CLASS

### ISSUE THREAD

**Issue by tilmantroester** [LSSTDESC/CCL#372]
At the moment CCL and CLASS seem to give different values (factor of ~2) for the energy density of relativistic neutrinos. 
Here's the snippet that should create the same cosmology but apparently doesn't:
```
import classy
import pyccl as ccl

class_params = {"output"    : "",
                "T_cmb"     : 2.725,
                "h"         : 0.7,
                "Omega_cdm" : 0.25,
                "Omega_b"   : 0.05,
                "Omega_k"  : 0.0,
                "N_ur"     : 3.0}
cosmo = classy.Class()
cosmo.set(class_params)
cosmo.compute()

ccl_params = ccl.Parameters(Omega_c=class_params["Omega_cdm"],
                            Omega_b=class_params["Omega_b"],
                            h=class_params["h"],
                            Omega_k=class_params["Omega_k"],
                            Neff=class_params["N_ur"],
                            A_s=2.0e-9,
                            n_s=1.0,
                            m_nu=0.0)

print("Omega_L: CLASS: {}, CCL: {}".format(cosmo.Omega_Lambda(), ccl_params.parameters.Omega_l))
print("Difference: {}".format(ccl_params.parameters.Omega_l-cosmo.Omega_Lambda()))
print("Fractional difference: {}".format(ccl_params.parameters.Omega_l/cosmo.Omega_Lambda()-1))
print()
print("Omega_g: CLASS: {}, CCL: {}".format(cosmo.Omega_g(), ccl_params.parameters.Omega_g))
print("Fractional difference: {}".format(ccl_params.parameters.Omega_g/cosmo.Omega_g()-1))
print()

Omega_ur = cosmo.get_background()["(.)rho_ur"]/cosmo.get_background()["(.)rho_crit"]
print("Omega_ur: CLASS: {}, CCL: {}".format(Omega_ur[-1], ccl_params.parameters.Omega_n_rel))
print("Difference: {}".format(ccl_params.parameters.Omega_n_rel-Omega_ur[-1]))
print("Fractional difference: {}".format(ccl_params.parameters.Omega_n_rel/Omega_ur[-1]-1))
```
This yields the following output for me:
```
Omega_L: CLASS: 0.6999152076713755, CCL: 0.6999327314693308
Difference: 1.75237979552767e-05
Fractional difference: 2.5037029862007287e-05

Omega_g: CLASS: 5.043194045865188e-05, CCL: 5.04319404656203e-05
Fractional difference: 1.3817480493116818e-10

Omega_ur: CLASS: 3.4360388165895934e-05, CCL: 1.683659020355884e-05
Difference: -1.7523797962337094e-05
Fractional difference: -0.5099999999339404
```

This difference in Omega_n_rel causes differences on the order of 1e-3 in the comoving distance at high redshift. Hacking Omega_n_rel to the CLASS value gets the agreement to 1e-5, which is what we want I think.

It's also entirely possible that I'm using either software wrong, in which case this can be closed (but it would be great if someone could point out what I'm doing wrong).

**Comment by c-d-leonard** [comment]
Hi @tilmantroester, thanks for bringing this up. I will check into this.

**Comment by tilmantroester** [comment]
Addressed with PR #374.
